syntax = "proto3";

import "rpc/common.proto";


service TransformLocation {
  rpc ToXY(LatitudeLongitude) returns (XY);
  rpc ToLatitudeLongitude(XY) returns (LatitudeLongitude);
}
